selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -82,6 +82,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(SeleniumTestHandler.class);
   private static final AtomicBoolean isCleanUpCompleted = new AtomicBoolean();
+  private static final AtomicBoolean isWebDriverSessionCreationChecked = new AtomicBoolean();
```

Comment:
If we invoke `checkWebDriverSessionCreation` in constructor then we won't needed this flag.

Reply:
Good catch. Thank you!